csharp/branches-quickstart/README.md

```diff
@@ -0,0 +1,22 @@
+C# Branches and Loops Sample
+================
+
+This sample is part of the [Branches and Loops tutorial](https://docs.microsoft.com/en-in/dotnet/csharp/tutorials/intro-to-csharp/branches-and-loops-local)
```

Comment:
We remove the locale from URLs so that the links localize for readers in different cultures or regions:

```suggestion
This sample is part of the [Branches and Loops tutorial](https://docs.microsoft.com/dotnet/csharp/tutorials/intro-to-csharp/branches-and-loops-local)
```